src/test/resources/com/puppycrawl/tools/checkstyle/astprinter/expectedInputJavadocComment.txt

```diff
@@ -1,25 +1,25 @@
-JAVADOC -> <p>\r\nMy {@code Class}\r\n</p>\r\n@see never<EOF> [0:0]
```

Comment:
I think we expect no change in this file for column numbers.
If you look at test and input that uses this file, the actual input is just pure Javadoc (no `/**` or `*/`). The column/line numbers should match up to it exactly, imo.
We could correct `DetailNodeTreeStringPrinter.createFakeBlockComment` for this change by hacking their column numbers to -3 and -1 so input starts exactly at 0.
@romani You agree?